ui/src/pages/Input/InputPage.tsx

```diff
@@ -259,6 +313,7 @@ function InputPage() {
                                             handleRequestOpen({ serviceName: service.name })
                                         }
                                         handleOpenPageStyleDialog={handleOpenPageStyleDialog}
+                                        displayActionBtnAllRows
```

Comment:
seems like this changes causes a logic change. Why it was added?

Reply:
this is for type check workaround